#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_Util.dll.config.dpatch by Mirco Bauer <meebey@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add dllmaps for Util.dll

@DPATCH@
diff -urNad beagle-0.3.9~/Util/Util.dll.config beagle-0.3.9/Util/Util.dll.config
--- beagle-0.3.9~/Util/Util.dll.config	2008-04-14 08:16:47.000000000 +1000
+++ beagle-0.3.9/Util/Util.dll.config	2009-02-24 12:16:07.994503413 +1100
@@ -1,2 +1,6 @@
 <configuration>
+	<dllmap dll="libc" target="libc.so.6"/>
+	<dllmap dll="libbeagleglue" target="/usr/lib/beagle/libbeagleglue.so"/>
+	<dllmap dll="libbeagleglue.so" target="/usr/lib/beagle/libbeagleglue.so"/>
+	<dllmap dll="libexif.dll" target="libexif.so.12"/>
 </configuration>
